src/components/Hero/Hero.module.scss

```diff
@@ -85,3 +85,49 @@
   height: 100%;
   background-color: rgba(0, 0, 0, 0.3);
 }
+
+@media (max-width: 600px) {
+  .heroFooter {
+    min-width: 100vw;
+    font-size: 20vw;
+    gap: 1vh;
+    right: 0;
+    flex-direction: column;
+    bottom: 14;
+    padding: 0 1vw 2vh 0;
+  }
+
+  .heroType {
+    width: 95vw;
+    height: 9vh;
+  }
+
+  .header {
+    position: fixed;
+    top: 0;
+  }
+
+  .imageWrapper {
+    overflow: hidden;
+    max-height: 100vh;
+    height: 100vh;
+    max-width: 100vw;
+  }
+
+  .logoH1,
+  .logoH2 {
+    width: 50vw;
```

Comment:
these shouldnt need this width. we should use justify-content: space between around the logo/menu